modules/op-installing-results.adoc

```diff
@@ -0,0 +1,64 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-chains-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="installing-results_{context}"]
+= Installing {tekton-results}
+
+[role="_abstract"]
+To install {tekton-results}, you must provide the required resources and then create and apply a `TektonResult` custom resource (CR). The {pipelines-shortname} Operator installs the Results services when you apply the `TektonResult` custom resource.
+
+.Procedure
+
+. Create a secret with the database user named `result` and a random password in the `openshift-pipelines` namespace by entering the following command:
++
+[source,bash]
```

Comment:
```suggestion
[source,terminal]
```
Global comment to all similar instances.
@mramendi: Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting

For Bash "here" documents use [source,terminal].